system-test/spanner.ts

```diff
@@ -4405,6 +4405,31 @@ async function deleteTestInstances() {
   );
 }
 
+async function deleteOldTestInstances() {
+  const [instances] = await spanner.getInstances();
+  // Leave only instances that contain PREFIX in their name
+  // and where created more that an hour ago.
+  const toDelete = instances.filter(
+    instance =>
+      instance.id.includes(PREFIX) &&
+      (Math.round(Date.now() / 1000) -
+        Number(instance.metadata.labels.created)) /
+        (60 * 60) >
+        1
+  );
+
+  const limit = pLimit(5);
+  return Promise.all(
+    toDelete.map(instance =>
+      limit(() =>
+        setTimeout(() => {
+          instance.delete();
+        }, 500)
```

Comment:
Maybe move to a variable whose name also explains why the delay is necessary.

Reply:
Refactored and clarified as per original [comment](https://github.com/googleapis/nodejs-spanner/blob/bf778cc0936c16ee0ef7b396c4698c2a1df30ce5/system-test/spanner.js#L4079)